HLTrigger/Muon/plugins/HLTMuonDimuonL3Filter.cc

```diff
@@ -595,17 +602,21 @@ bool HLTMuonDimuonL3Filter::applyDiMuonSelection(const RecoChargedCandidateRef&
   double pt12 = p.pt();
   LogDebug("HLTMuonDimuonL3Filter") << " ... 1-2 pt12= " << pt12;
 
+  // Angle between the muons
+  if (deltaR(p1, p2) < min_DiMuAngle_)
```

Comment:
```suggestion
  if (applyDiMuonDeltaRCut_ and reco::deltaR2(p1, p2) < min_DiMuonDeltaR2_)
```

Reply:
The default value of the min_DiMuonDeltaR2 ensures when someone doesn't want the deltaR cut, they do not need to change the value or assign a value to this value. Adding another parameter to this list doesn't add any extra function. This is the same philosophy as the other cuts placed in this module. Applying the new parameter also adds to the time required to run this module. Please correct me if I am missing something.

Reply:
> Adding another parameter to this list doesn't add any extra function.

It improves the readability of the code, clarifying when the DeltaR cut will be applied. Also, if someone uses for example `"MinDiMuonDeltaR" = cms.double(-1.)` in the python configuration, he/she would rightfully expect that the DeltaR cut won't be applied (this is how other modules usually behave), but then `min_DiMuonDeltaR2_` would be `== -1*-1 == 1`, and without `applyDiMuonDeltaRCut_ and [..]` one would just end up cutting on `DR2 < 1` unintentionally.

> Applying the new parameter also adds to the time required to run this module.

Not that I have done any measurements, but I doubt that `true and "calculationX"` is significantly slower than just `"calculationX"`. In fact, it's generally the opposite; in the majority of cases, modules will have "applyDiMuonDeltaRCut_ == false", and in that case `deltaR2` won't be calculated in the first place, saving time (however small this effect is).

Reply:
I understand now. Thank you for the review. I will accept the commits relevant to the DeltaR2 cut.